packages/core/contracts/financial-templates/test/ExpiringMultiPartyMock.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "../../common/implementation/FixedPoint.sol";
+
+
+interface FinancialProductLibrary {
```

Comment:
Maybe I'm missing something obvious but why aren't you using the actual `financial-templates/common/financial-product-libraries/FinancialProductLibrary.sol` contract here?

Reply:
No real reason. I've updated to use that import.